97.md

```diff
@@ -0,0 +1,82 @@
+NIP-97
+======
+
+Event Recommendation Services
+-----------------------------
+
+`draft` `optional` `author:vitorpamplona`
+
+## Abstract
+
+Service providers want to offer event recommendations to users that subscribe to them. Specifics of the recommendation algorithms will differ between event kinds and provider implementations. This NIP describes a general framework for subscribing and performing recommendation requests.
+
+# Recommendation Subscriptions list
+
+A special event with kind `10020` "Recommendation Subscription List" is defined as a [NIP-16 - Replaceable Events](16.md) of public and/or private `p` tags, one for each of the subscribed services the author intends to use. Each tag entry should contain the public key for the service provider and a relay URL where the provider hosts the recommendation events. i.e., `["p", <32-bytes hex key>, <main relay URL>]`.
+
+For example:
+
+```json
+{
+  "kind": 10020,
+  "tags": [
+    ["p", "91cf9..4e5ca", "wss://provider1.com/"],
+    ["p", "14aeb..8dad4", "wss://provider2.com/nostr"],
+    ["p", "612ae..e610f", "ws://provider3.com/ws"]
+  ],
+  "content": "<private list>",
+  ...other fields
+}
+```
+
+Providers can offer as many algorithms as they way as long as they offer one per public key. 
+
+# Recommendation Request
+
+Clients can request recommendations at will using a kind `20020` ephemeral event. `p` tags MUST contain the group of users to make the recommendation for as well as the pub key of the service provider. The `.content` field MUST contains an array of NOSTR `REQ` filters ([NIP-1 - Basic Protocol](1.md)) that describe the type and constraints of the requested recommendation events.
+
+```json
+{
+  "kind": 20020,
+  "tags": [
+    ["p", "<service pubkey>"],
+    ["p", "91cf9..4e5ca", "wss://relay1.com"],
+  ],
+  "content": "[<FILTER1>, <FILTER2>, ...]",
```

Comment:
Should this be an object w/ 'filters' array to allow for further extensibility? I miss that in the REQ request, let's not re-create the issue here.